src/libmongoc/tests/test-libmongoc.c

```diff
@@ -2477,6 +2477,7 @@ WIRE_VERSION_CHECKS (6)
 WIRE_VERSION_CHECKS (7)
 WIRE_VERSION_CHECKS (8)
 WIRE_VERSION_CHECKS (9)
+WIRE_VERSION_CHECKS (12)
```

Comment:
4.4 is wire version 9 and 5.0 jumps right to 12 (see: [wireversion-featurelist.rst](https://github.com/mongodb/specifications/blob/master/source/wireversion-featurelist.rst)). Let me know if this warrants a comment, but I imagine these only get created when they're actually needed by a test so I wouldn't anticipate this being an issue in the future.

Reply:
Minor: [wire_version.h|https://github.com/mongodb/mongo/blob/2d0a51a52385cb01e4bba0170270daaa349cc99b/src/mongo/db/wire_version.h#L96] notes wire version 12 is for 4.9, and 13 for 5.0.
```suggestion
WIRE_VERSION_CHECKS (13)
```

Though [SERVER-20547](https://jira.mongodb.org/browse/SERVER-20547) was implemented in 4.9, I don't think we'll want to switch on whether a server is >= 4.9.